sources/web/datalab/polymer/components/directory-picker-dialog/directory-picker-dialog.ts

```diff
@@ -81,6 +92,21 @@ class DirectoryPickerDialogElement extends BaseDialogElement {
     return this._memoizedTemplate;
   }
 
+  ready() {
+    if (!this.readyPromise) {
+      super.ready();
+      this.readyPromise = this.$.filePicker.ready();
+      this.addEventListener('iron-overlay-opened',
+        () => { this.resizeHandler(); });
+    }
+    return this.readyPromise;
+  }
+
+  resizeHandler() {
```

Comment:
Do we need this handler, or can we just call the resizeHandler of the picker in the event listener above?

Reply:
I think having a resize handler here makes it more clear as to what the intent is, and is less likely to get overlooked if someone needs to do something else on a resize event.